packages/components/releaseNotes/components.md

```diff
@@ -1,6 +1,11 @@
 # @labkey/components
 Components, models, actions, and utility functions for LabKey applications and pages.
 
+### version 1.19.x
```

Comment:
We don't exactly have a convention for this yet, but I think we should somehow mark any release notes items that affect the "public API" with some sort of annotation so that we can find them more quickly. Thoughts? Maybe something like `### version 1.19.x (Public API update)`?

Reply:
I've added "Public API update" below the version line.
I'll update the version from 'x' to an actual version num before merging.